ee/packages/presence/src/Presence.ts

```diff
@@ -68,7 +68,7 @@ export class Presence extends ServiceClass implements IPresence {
 		try {
 			await Settings.updateValueById('Presence_broadcast_disabled', false);
 
-			this.hasLicense = await License.hasModule('scalability');
+			this.hasLicense = await License.hasModule('unlimited-presence');
```

Comment:
Not sure the cleanest way here.  But if accepting the other to prevent breaking changes then would need to accept either here as well